addons/website_slides/static/src/js/slides.js

```diff
@@ -147,43 +162,77 @@ $(document).ready(function () {
                 success: callback
             });
         };
-        var current_url = window.location.origin + window.location.pathname;
-        socialgatter('https://www.linkedin.com/countserv/count/share?url=', current_url, function (data) {
-            $('#linkedin-badge').text(data.count || 0);
-            $('#total-share').text(parseInt($('#total-share').text()) + parseInt($('#linkedin-badge').text()));
-        });
-        socialgatter('https://cdn.api.twitter.com/1/urls/count.json?url=', current_url, function (data) {
-            $('#twitter-badge').text(data.count || 0);
-            $('#total-share').text(parseInt($('#total-share').text()) + parseInt($('#twitter-badge').text()));
-        });
-        socialgatter('https://graph.facebook.com/?id=', current_url, function (data) {
-            $('#facebook-badge').text(data.shares || 0);
-            $('#total-share').text(parseInt($('#total-share').text()) + parseInt($('#facebook-badge').text()));
+        var update_statistics = function(social_site, slide_url) {
+            if (social_site == 'linkedin') {
+                socialgatter(social_urls['linkedin'], slide_url, function(data) {
```

Comment:
can not make common method like

def set_share_text(social_site){
    socialgatter(social_urls[social_site], slide_url, function(data) {
                    var $el_social = $('#%s-badge'%social_site)
                    $el_social.text(data.count || 0);
                    $('#total-share').text(parseInt($('#total-share').text()) + parseInt($el_social.text()));
    });
}